extension/main.js

```diff
@@ -16,13 +16,31 @@
 	}
 
 	function update() {
-		gitHubNotifCount(function (count) {
+		gitHubNotifCount(function (count, interval) {
+			var period = 1;
+			if (interval !== GitHubNotify.settings.get('interval')) {
+				GitHubNotify.settings.set('interval', interval);
+				period = Math.ceil(interval / 60);
+				if (period < 1) {
```

Comment:
```js
if (period < 1 || isNaN(period)) {
```

Otherwise I get this error:

```
extensions::uncaught_exception_handler:8 Error in response to alarms.clearAll: Error: Invalid value for argument 1. Property 'periodInMinutes': Value must not be NaN.
    at Object.callback (chrome-extension://nodblbbnmnhfphpaokabkcinaglkedik/main.js:28:20)
    at chrome-extension://nodblbbnmnhfphpaokabkcinaglkedik/main.js:27:19
    at window.gitHubNotifCount (chrome-extension://nodblbbnmnhfphpaokabkcinaglkedik/api.js:68:4)
    at update (chrome-extension://nodblbbnmnhfphpaokabkcinaglkedik/main.js:19:3)
    at chrome-extension://nodblbbnmnhfphpaokabkcinaglkedik/main.js:76:2
    at chrome-extension://nodblbbnmnhfphpaokabkcinaglkedik/main.js:77:3
```